new-starter-guide/2016-03-15-new-starter-to-origami.md

```diff
@@ -0,0 +1,134 @@
+# New starter guide
+
+Welcome to Origami!
+
+## Checklist of things to do
+
+The checklist assumes that you have done the following:
+
+1. Visit IT Service Desk to get your computer account up and running.
+2. Logged on your own work machine and email account.
+
+### Admin privileges on your machine
+
+IT Services should have enabled admin privileges when they have set up your machine but you should [request admin access to your FT laptop via Salesforce](https://financialtimes.my.salesforce.com/home/home.jsp) to ensure it does not get revoked.
+
+If you do not have admin privileges then email `itservicedesk@ft.com`. Include your desk number (which is usually on the left of the desk) and the machine number with the email. You also will need to say why you need admin privileges, or alternatively copy the template below and edit accordingly.
```

Comment:
"usually on the left of your desk" can probably be something like "usually on a label at the front of your desk"

(they're on the outer corner of these 'dual-person' desks, but could be somewhere else on others)